### PR TITLE
submitting multiply function

### PR DESCRIPTION
**Pull request by Debebe**

### CODE REVIEW

File: pythoncalculator/__init__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
-from .add import add
```
Comment (claraqmdc) [inline pythoncalculator/__init__.py]: you need to keep this line of code

File: pythoncalculator/multiply.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```
Comment (claraqmdc) [inline pythoncalculator/multiply.py]: it should be a * and not a +

```suggestion
    return x * y
```

**Review by claraqmdc — CHANGES_REQUESTED** [review]

File: pythoncalculator/__init__.py (hunk 1 above)
Comment (Debebe) [reply pythoncalculator/__init__.py]: Thanks, nice catch

**Review by claraqmdc — APPROVED** [review]
Sounds good!

**Pull request merged**